tests/cicd/src/scripts/get_data_for_builds.sh

```diff
@@ -50,4 +50,4 @@ python prepare_dutchf3.py split_train_val patch   --data_dir=${DATA_F3} --label_
 DATA_CHECKERBOARD="${DATA_CHECKERBOARD}/data"
 # repeat for checkerboard dataset
 python prepare_dutchf3.py split_train_val section --data_dir=${DATA_CHECKERBOARD} --label_file=train/train_labels.npy --output_dir=splits --split_direction=both
-python prepare_dutchf3.py split_train_val patch   --data_dir=${DATA_CHECKERBOARD} --label_file=train/train_labels.npy --output_dir=splits --stride=50 --patch_size=100 --split_direction=both
+python prepare_dutchf3.py split_train_val patch   --data_dir=${DATA_CHECKERBOARD} --label_file=train/train_labels.npy --output_dir=splits --stride=50 --patch_size=100 --split_direction=both --section_stride=100
```

Comment:
The section stride is very big. This might lead to a really small training set.. it will make training faster, but if we are testing the performance of this model, it might note perform well at all.

Reply:
@yalaudah I mark this as resolved per our chat :)